src/config/containers-and-vms/libvirt.md

```diff
@@ -0,0 +1,28 @@
+# libvirt
+
+[libvirt](https://libvirt.org/) is an API and daemon for managing platform
+virtualization, supporting virtualization technologies such as LXC, KVM, QEMU,
+Bhyve, Xen, VMWare, and Hyper-V.
+
+To use libvirt, install the `libvirt` package and
+[enable](../services/index.md#enabling-services) the `libvirtd`, `virtlockd` and
+`virtlogd` services. The `libvirtd` daemon can be reconfigured at runtime via
+[virt-admin(1)](https://man.voidlinux.org/virt-admin.1).
+
+The `libvirt` package provides the [virsh(1)](https://man.voidlinux.org/virsh.1)
+interface to libvirtd. `virsh` is an interactive shell and batch-scriptable tool
+for performing management tasks, including creating, configuring and running
+virtual machines, and managing networks and storage. Note that `virsh` usually
+needs to be run as root, as described in the `virsh` man page:
+
+> Most virsh commands require root privileges to run due to the communications
+> channels used to talk to the hypervisor. Running as non root will return an
+> error.
```

Comment:
This is true, but `libvirt` ships with a polkit rule `/usr/share/polkit-1/rules.d/50-libvirt.rules` that grants management privileges to users of the `libvirt` group. Thus, I suggest you add something like the following text:
```
As an alternative, if you have the `polkit` and `dbus` packages installed and you enable the `dbus` service, `libvirtd` will grant necessary privileges to any user added to the `libvirt` group.
```

Reply:
Thanks, PR updated.